src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, sans-serif;
+  --link-hover-active: #00acdc;
+}
+
+body,
+div,
+ul,
+li,
+a {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: block;
+  width: 40px;
+  height: 40px;
```

Comment:
It would be great to create variable for all main values and colors

Reply:
Not sure I can figure out main values in this task. Would you please give me either refference or list of main values for this task

Reply:
That might be smth like `--logo-size: 40px`, but not critical IMO